src/translator/byte_array_util.cpp

```diff
@@ -5,6 +5,75 @@
 namespace marian {
 namespace bergamot {
 
+// This is a basic validator that checks if the file has not been truncated
+// it basically loads up the header and checks
+
+// This struct and the getter are copied from the marian source, because it's located
+// inside src/common/binary.cpp:15 and we can't include it.
```

Comment:
/// to be picked up by doxygen, consistent with what exists in source.

Reply:
I don't want this to be picked up by doxygen? It's an internal function, not exposed by the headers and only really meant to be read by the developers.

Reply:
Anonymous namespace please.  

The standard for documentation is higher in this repo than it is elsewhere, sorry @XapaJIaMnu.

Reply:
Sorry, I genuinely do not understand how I should document this better. I'm went through the repo and looked at various files: (this one comes to mind) https://github.com/browsermt/bergamot-translator/blob/main/src/translator/request.h

Do I need to add the `defines` bit on top?

Reply:
I think what @kpu means is to either leave it in an anonymous namespace if it's local and don't bother documenting, or fit it with doxygen compatible documentation.

[This](https://github.com/browsermt/bergamot-translator/blob/jp/response-builder/src/translator/request.h) is the request which is documented. The target is a sphinx doc build, a demo of which looks like [this](https://jerinphilip.github.io/bergamot-translator/api/namespace_marian__bergamot.html).

Reply:
Ok that should be fixed now.